docs/layouts/shortcodes/ebnf.html

```diff
@@ -0,0 +1,66 @@
+{{ $statements :=  .Inner }}
+{{ $type := default "tabs" (.Get "type") }}
+{{ $statements = replaceRE " *- *" "" $statements}}
+{{ $statements = replaceRE "\n+" "," $statements}}
+{{ $statements = replaceRE " +" "" $statements}}
+{{ $statements = replaceRE ",+" "," $statements}}
```

Comment:
it may be better to not allow syntax like ",,"
```suggestion
```

Reply:
Multiple whitespaces and newlines are converted to commas. This could lead to multiple commas during input cleaning. If we decide on a  strict format, this wont be needed